File: UI/lib/shared/models/requests/location_update_request.dart
  class LocationUpdateRequest {

    final String? name;
    final String? country;
    final String? region;

    const LocationUpdateRequest({
      this.name,
      this.country,
      this.region,
    });

    Map<String, dynamic> toJson() => {
          'name': name, 
          'country': country,
          'region': region,
        };
  }

File: backend/TenderGo.Models/Requests/LocationUpdateRequest.cs
using System.ComponentModel.DataAnnotations;

public class LocationUpdateRequest
{
    [Required]
    [MaxLength(100)]
    public string Name { get; set; } = string.Empty;

    [Required]
    [MaxLength(100)]
    public string Country { get; set; } = string.Empty;

    [MaxLength(100)]
    public string? Region { get; set; }
}